pkg/ccl/backupccl/backup_destination.go

```diff
@@ -120,13 +130,52 @@ func resolveDest(
 			return "", "", "", nil, nil, err
 		}
 		if exists {
-			// The backup in the auto-append directory is the full backup.
+			// The backup in defaultStore is the full backup.
+			// This would only occur if:
+			//  - User specified 'BACKUP INTO IN LATEST'
+			//  - User specified a subdir via `BACKUP <targets...> INTO [<subdir... > IN] <destination>`
+			//    that contains a full backup
+			//  - User specified a <destination> via `BACKUP <targets...> TO <destination...>` that
+			//    contains a full backup
+
+			// add the full backup URI
 			prevBackupURIs = append(prevBackupURIs, defaultURI)
-			priors, err := FindPriorBackups(ctx, defaultStore, OmitManifest)
+
+			var priors []string
+			backupChainURI := defaultURI
+			if len(incrementalStorage) > 0 {
+				// Implies the incremental backup chain lives in
+				// incrementalStorage/chosenSuffix, while the full backup lives in
+				// defaultStore/chosenSuffix. The incremental backup chain in
+				// incrementalStorage/chosenSuffix will not contain any incremental backups that live
+				// elsewhere.
+				incrementalURI, _, err := getURIsByLocalityKV(incrementalStorage,
```

Comment:
what is `getURIsByLocalityKV` doing for us here, if we're throwing away the returned map? just path joining chosenSuffix to incrementalStorage?

Reply:
hehe. i was being lazy. There's some funky parsing commands required (you cant just join the path) and I wanted to abstract away from that. I changed this line to use the lighter weight `getLocalityAndBaseURI`